package.json

```diff
@@ -214,7 +214,7 @@
 		"changelog": "./bin/plugin/cli.js changelog",
 		"check-licenses": "concurrently \"wp-scripts check-licenses --prod --gpl2 --ignore=@react-native-community/cli,@react-native-community/cli-platform-ios\" \"wp-scripts check-licenses --dev\"",
 		"precheck-local-changes": "npm run docs:build",
-		"check-local-changes": "( git diff -U0 | xargs -0 node bin/process-git-diff ) || ( echo \"There are local uncommitted changes after one or both of 'npm install' or 'npm run docs:build'!\" && exit 1 );",
+		"check-local-changes": "( git diff -U0 | xargs -0 node bin/process-git-diff ) || ( echo \"There are local uncommitted changes after one or both of 'npm install' or 'npm run docs:build'!\" && git diff --exit-code && exit 1 );",
```

Comment:
Why `--exit-code`? From the documentation, I'm not sure how it helps, as we don't look at exit codes (and actually it could short-circuit that last `&& exit 1`.

> **--exit-code**
> Make the program exit with codes similar to diff(1). That is, it exits with 1 if there were differences and 0 means no differences.

Reply:
@mcsf https://github.com/WordPress/gutenberg/pull/24154#discussion_r462040410

When I tested, `git diff` doesn't exit, it awaits user input.

Reply:
Hm, I see. What about `git --no-pager diff`?

Reply:
Did not know about that option, that looks like it'd be better, yep 👍